### ISSUE TITLE
Discord/Battle tag mandatory fields

### ISSUE THREAD

**Issue by Tueri** [Icecrown-Community/ICC-Member-Website-Public#70]
Providing one OR the other should be mandatory, else the customer is not contactable and advertisers get a bit rowdy when they miss their invites if name-realm is not spelled correctly or similar. It would also promote Gluth messages to customers.

**Comment by TheAmazingTyr** [comment]
Hey there! I'd like to butt in on this one, there will inevitably be some buyers who do not wish to share other ways of contact, idea:
a checkbox that customer did not want to provide alternative contact info. This should however come at an increased deposit and forfeiting an option to get refunded if we cannot make contact via whispers. this should be VERY CLEARLY explained by the advertiser
In case that the advertiser "cba", and abuses the checkbox to not have to provide info, bowling.

**Comment by Tueri** [comment]
a checkbox for such things would be fine also, as long as it prevents info from being added to the fields if checked, and doesn't feed data from those fields (if any was entered) before the box was ticked, maybe even have a warning prompt too.

**Comment by iElsha** [comment]
TLDR;

Add mandatory contact options (Discord, battle.net). If the customer doesn't want to share we respect the decision through a checkbox for the advertiser.

**Comment by iElsha** [comment]
@Yoldrim reach @IndigoICC for more information if needed

**Comment by Dejnaa** [comment]
Mr @Yoldrim any ETA on this?

**Comment by Bgabor997** [comment]
I finished the testing for it yesterday

**Comment by Dejnaa** [comment]
@Bgabor997 Awesome, I can always count on you <3 
So I assume this is now on Elshas table to go live when he is back from vacation?

**Comment by Bgabor997** [comment]
i can deploy if he approves

**Comment by Dejnaa** [comment]
@iElsha Please approve

**Comment by iElsha** [comment]
Merged 4 days ago already
